src/foremast/utils/templates.py

```diff
@@ -16,6 +16,7 @@
 """Render Jinja2 template."""
 import logging
 import os
+import pathlib
```

Comment:
Do we have pathlib in requirements.txt?

Reply:
Ah I did not know that. But if we merge that, will it break Python2 users then? How do we keep foremast compatible with Python2?

Reply:
We never supported python 2.
https://github.com/gogoair/foremast/blob/master/setup.py#L45

Reply:
It if worked, that's great but it was not intentional.